src/ClimaLSM.jl

```diff
@@ -1,24 +1,21 @@
 module ClimaLSM
-###What is the role of configurations here?
 using UnPack
 using ClimaCore
 import ClimaCore: Fields
 include("Domains.jl")
 using .Domains
-include("Configurations.jl")
-using .Configurations
```

Comment:
maybe we should just delete the Configurations file for now, nothing in it really.

Reply:
Fine either way